js/app/grid.js

```diff
@@ -354,6 +354,20 @@ function(config, Phaser, music){
     }
 
     /**
+     * Clear all blocks on the grid.
+     */
+    Grid.prototype.clearAll = function() {
```

Comment:
Perhaps this should be called 'destroyAll', as it does not invoke clear on the blocks.